### PR TITLE
feat(perception_launch): add launcher for pointpainting

### PR DESCRIPTION
**Pull request by tzhong518**
Signed-off-by: tzhong518 <sworgun@gmail.com>

## PR Type

- New Feature

## Description

Add launcher for pointpainting in camera_lidar_fusion_based_detection

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: perception_launch/launch/object_recognition/detection/detection.launch.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <!-- lidar param -->
   <arg name="input/pointcloud"/>
-  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`"/>
+  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`, `pointpainting`"/>
```
Comment (yukke42) [inline perception_launch/launch/object_recognition/detection/detection.launch.xml:7]: Cloud you change other files s.t. camera_lidar_fusion_based_detection.launch.xml or perception.launch.xml in the same way?

File: perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 2]
```diff
@@ -181,6 +181,40 @@
     </group>
   </group>
 
+  <!-- PointPainting -->
+  <group if="$(eval &quot;'$(var lidar_detection_model)'=='pointpainting'&quot;)">
+    <!-- <push-ros-namespace namespace="pointpainting"/> -->
```
Comment (yukke42) [inline perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml]: This comment out is as you expected?

**Comment by yukke42** [comment]
Cloud you change pointpainting_launch to perception_launch of the tilte?

File: perception_launch/launch/object_recognition/detection/detection.launch.xml (hunk 1 above)
Comment (tzhong518) [reply perception_launch/launch/object_recognition/detection/detection.launch.xml:7]: https://github.com/tier4/autoware_launch/pull/426/commits/3bf7e891e0fe0060395bc80e82b918aa291244a9

File: perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml (hunk 2 above)
Comment (tzhong518) [reply perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml]: I cancelled the comment out.

**Comment by yukke42** [comment]
This PR requires <https://github.com/tier4/autoware_launch/pull/449>.

**Review by yukke42 — APPROVED** [review]
LGTM

**Comment by tkimura4** [comment]
@miursh @yukke42 @tzhong518 
 Is it okay to merge this PR?

**Pull request merged**